Dockerfile

```diff
@@ -1,9 +1,13 @@
 FROM python:2.7
-RUN groupadd -g 10001 app && useradd -d /app -g 10001 -G app -M -s /bin/sh -u 10001 app
+ENV NEWRELIC_LIC_KEY ${NEWRELIC_LIC_KEY:-"1234"}
+ENV ENABLE_NEWRELIC $ENABLE_NEWRELIC
+RUN groupadd -g 10001 app && \
+      useradd -d /app -g 10001 -G app -M -s /bin/sh -u 10001 app
```

Comment:
Today I was reminded of the -U flag which will create the group automagically. winning.